src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -446,10 +474,7 @@ export async function singleCheckBox(payload) {
 export function toggleCheckBox(payload) {
   return () => {
     const currentAllFilterVariables = payload === {} ? allFilters : createFilterVariables(payload);
-    // For performance issue we are using initial dasboardquery instead of fitered for empty filters
-    if (_.isEqual(currentAllFilterVariables, allFilters())) {
```

Comment:
Is this removed on purpose? As clearing all we dont want to send a query with empty params?